template/template_test.go

```diff
@@ -172,6 +172,71 @@ func Test_ExecuteComplexMessage(t *testing.T) {
 	}
 }
 
+func Test_Matcher(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name      string
+		inputStr  string
+		inputMaps []map[string]any
```

Comment:
Alternative names for `inputStr`: `text`, `msg`, `message`.

```suggestion
		text  string
		inputs []map[string]any
```